### ISSUE TITLE
Holorchis castex links to EOL id but nomer/taxon graph have many more links

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/globalbioticinteractions#448]
When visiting https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=Holorchis%20castex , the attached screenshot was taken. 

![Screenshot from 2020-01-17 09-55-31](https://user-images.githubusercontent.com/1084872/72634599-b6a85800-390f-11ea-8b28-c7fab0a1bd55.png)

When running nomer using taxon graph 0.3.x against the name using, the following was seen:

```
$ echo -e "\tHolorchis castex" | nomer append globi-cache
	Holorchis castex	SAME_AS	EOL_V2:11987314	Holorchis castex	species		Animalia | Platyhelminthes | Trematoda | Plagiorchiida | Lepocreadiidae | Holorchis | Holorchis castex	EOL:1 | EOL:2884 | EOL:2894 | EOL_V2:2976 | EOL_V2:2995 | EOL_V2:2951012 | EOL_V2:11987314	kingdom | phylum | class | order | family | genus | species	https://doi.org/10.5281/zenodo.1495266#11987314	
	Holorchis castex	SAME_AS	GBIF:5890922	Holorchis castex	species		Animalia | Platyhelminthes | Trematoda | Plagiorchiida | Lepocreadiidae | Holorchis | Holorchis castex	GBIF:1 | GBIF:108 | GBIF:345 | GBIF:434 | GBIF:6420 | GBIF:4346132 | GBIF:5890922	kingdom | phylum | class | order | family | genus | species	http://eol.org/pages/11987314	
	Holorchis castex	SAME_AS	IRMNG:11821202	Holorchis castex	species		Animalia | Platyhelminthes | Trematoda | Plagiorchiida | Lepocreadiidae | Holorchis | Holorchis castex	IRMNG:11 | IRMNG:116 | IRMNG:1259 | IRMNG:11762 | IRMNG:113051 | IRMNG:1369632 | IRMNG:11821202	kingdom | phylum | class | order | family | genus | species	http://eol.org/pages/11987314	
	Holorchis castex	SAME_AS	NCBI:681679	Holorchis castex	species		| Eukaryota | Opisthokonta | Metazoa | Eumetazoa | Bilateria | Platyhelminthes | Trematoda | Digenea | Plagiorchiida | Lepocreadiata | Lepocreadioidea | Lepocreadiidae | Holorchis | Holorchis castex	NCBI:131567 | NCBI:2759 | NCBI:33154 | NCBI:33208 | NCBI:6072 | NCBI:33213 | NCBI:6157 | NCBI:6178 | NCBI:6179 | NCBI:27871 | NCBI:1776234 | NCBI:27873 | NCBI:27875 | NCBI:681678 | NCBI:681679	|  |  | kingdom |  |  | phylum | class | subclass | order | suborder | superfamily | family | genus | species	http://eol.org/pages/11987314	
	Holorchis castex	SAME_AS	OTT:627299	Holorchis castex	species		Animalia | Platyhelminthes | Trematoda | Plagiorchiida | Lepocreadiidae | Holorchis | Holorchis castex	IRMNG:11 | IRMNG:116 | IRMNG:1259 | IRMNG:11762 | IRMNG:113051 | IRMNG:1369632 | IRMNG:11821202	kingdom | phylum | class | order | family | genus | species	http://eol.org/pages/11987314	
	Holorchis castex	SAME_AS	WD:Q41000883	Holorchis castex	species		| Holorchis castex	WD:Q18583761 | WD:Q41000883	| species	https://www.wikidata.org/wiki/Q41000883	
	Holorchis castex	SAME_AS	WORMS:594684	Holorchis castex	species		Biota | Animalia | Platyhelminthes | Neodermata | Trematoda | Digenea | Plagiorchiida | Lepocreadiata | Lepocreadioidea | Aephnidiogenidae | Holorchis | Holorchis castex	WORMS:1 | WORMS:2 | WORMS:793 | WORMS:853123 | WORMS:19948 | WORMS:108400 | WORMS:108402 | WORMS:468920 | WORMS:108408 | WORMS:744186 | WORMS:108572 | WORMS:594684	| kingdom | phylum | subphylum | class | subclass | order | suborder | superfamily | family | genus | species	http://eol.org/pages/11987314	
```

Expected: 
search result includes Holorchis castex with multiple taxon page links.

Actual:
search results include single EOL Holorchis castex link.

**Comment by jhpoelen** [comment]
Root cause of this issue is that the additional taxa are being added based on the id they are being mapped into, not the id that was provided. In case of _Holorchis castex_, the provided id was EOL:11987314 and the resolved id was EOL_V2:11987314 (for EOL vs EOL_V2 prefix, see #403 ).

Because no EOL_V2 is not being provided by any source, adding of additional taxa fails. 

Potential solution - ensure to add additional taxa based on their original taxon id, not the one they happen to resolve to.

**Comment by jhpoelen** [comment]
After fixing the root cause and re-indexing of taxa, the expected taxonomic links related to _Holorchis castex_ appeared via GloBI search https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=Holorchis%20castex . See attached screenshot.

![Screenshot from 2020-03-13 07-30-09](https://user-images.githubusercontent.com/1084872/76630166-c8b40a80-64fc-11ea-966c-d3504c3d2ba9.png)
